app/src/main/java/org/mozilla/fenix/onboarding/HomeOnboardingDialogFragment.kt

```diff
@@ -37,7 +45,61 @@ class HomeOnboardingDialogFragment : DialogFragment() {
             context?.settings()?.let { settings ->
                 settings.hasShownHomeOnboardingDialog = true
             }
+            showJumpCFR()
             dismiss()
         }
     }
+
+    private fun showJumpCFR() {
+        val jumpBackInView = findJumpBackInView()
+        jumpBackInView?.let {
+            val crfDialog = createJumpCRF(anchor = jumpBackInView)
+            crfDialog.show()
+        }
+    }
+
+    private fun findJumpBackInView(): View? {
+        val list = activity?.findViewById<RecyclerView>(R.id.sessionControlRecyclerView)
+        val count = list?.adapter?.itemCount ?: return null
+
+        for (index in 0..count) {
+            val viewHolder = list.findViewHolderForAdapterPosition(index)
+            if (viewHolder is RecentTabsHeaderViewHolder) {
+                return viewHolder.containerView
+            }
+        }
+        return null
+    }
+
+    private fun createJumpCRF(anchor: View): Dialog {
+        val context: Context = requireContext()
+        val anchorPosition = IntArray(2)
+        val popupBinding = OnboardingJumpBackInCfrBinding.inflate(LayoutInflater.from(context))
+        val popup = Dialog(context)
+
+        popup.apply {
+            setContentView(popupBinding.root)
+            setCancelable(false)
+            // removing title or setting it as an empty string does not prevent a11y services from assigning one
+            setTitle(" ")
+        }
+        popupBinding.closeInfoBanner.setOnClickListener {
+            popup.dismiss()
+        }
+
+        anchor.getLocationOnScreen(anchorPosition)
+        val (x, y) = anchorPosition
+
+        popupBinding.root.measure(View.MeasureSpec.UNSPECIFIED, View.MeasureSpec.UNSPECIFIED)
+
+        popup.window?.let {
+            val attr = it.attributes
+            it.setGravity(Gravity.START or Gravity.TOP)
+            attr.x = x
+            attr.y = y - popupBinding.root.measuredHeight
+            it.attributes = attr
+            it.setBackgroundDrawable(ColorDrawable(Color.TRANSPARENT))
+        }
```

Comment:
non-blocking nit: Use `apply` when _applying_ changes to the item.
```suggestion
       popup.window?.apply {
            val attr = attributes
            setGravity(Gravity.START or Gravity.TOP)
            attr.x = x
            attr.y = y - popupBinding.root.measuredHeight
            attributes = attr
            setBackgroundDrawable(ColorDrawable(Color.TRANSPARENT))
        }
```